src/main/scala/org/allenai/wikitables/TestWikiTablesCli.scala

```diff
@@ -131,6 +142,44 @@ class TestWikiTablesCli extends AbstractCli() {
 
     denotations
   }
+
+  def answerSingleQuestion(modelFilename: String, options: OptionSet): Map[String, List[(Value, Double)]] = {
+    // TODO: Do not load model for each question. Different Cli?
+    val simplifier = ExpressionSimplifier.lambdaCalculus()
+    val comparator = new SimplificationComparator(simplifier)
+    val parser = loadSerializedParser(modelFilename)
+    val featureGenerator = parser.config.featureGenerator.get
+    val typeDeclaration = parser.config.typeDeclaration
+    val lfPreprocessor = parser.config.preprocessor
+
+    // Are there better alternatives than assigning the current timestamp as the exampleId?
```

Comment:
How about some hash of the question and the table string?  That'll make it easier to implement some kind of caching later, if we want to do that.